testkit/src/main/scala/org/bitcoins/testkit/BitcoinSAppConfig.scala

```diff
@@ -0,0 +1,103 @@
+package org.bitcoins.testkit
+
+import com.typesafe.config.Config
+import org.bitcoins.wallet.config.WalletAppConfig
+import org.bitcoins.node.config.NodeAppConfig
+import org.bitcoins.chain.config.ChainAppConfig
+import scala.concurrent.ExecutionContext
+import scala.concurrent.Future
+import java.nio.file.Files
+import com.typesafe.config.ConfigFactory
+
+/**
+  * A unified config class for all submodules of Bitcoin-S
+  * that accepts configuration. Thanks to implicit definitions
+  * in this case class' companion object an instance
+  * of this class can be passed in anywhere a wallet,
+  * chain or node config is required.
+  */
+case class BitcoinSAppConfig(confs: Config*) {
```

Comment:
So what is the implication here if you are only using a portion of bitcoin-s (i.e. wallet with no node, or chain with no node etc)? 

Maybe it doesn't matter and can be safely ignored?

I guess looking at this closer this is occurring in the testkit, so maybe it doesn't matter. But i would be nice to have something in src as well

Reply:
This is a very valid objection, but that's also why I placed it in testkit. Once we get some things going that are _not_ just tests I think we can come back and revisit this point, and then we hopefully know more about what requirements and project layout looks like